### PR TITLE
seismology: Add seismograph

### PR DESCRIPTION
**Pull request by core-man**
Addresses #7.

### CODE REVIEW

File: source/seismology/seismograph.rst [hunk 1]
```diff
@@ -1,3 +1,45 @@
 地震仪
 ======
 
+基本原理
+--------
+
+地震仪（seismograph）是检测和记录特别是由地震引起的地面质点运动（位移、速度、加速度、转动等）的仪器，
```
Comment (seisman) [inline source/seismology/seismograph.rst:7]:
```suggestion
地震仪（seismograph）是检测和记录地面质点运动（位移、速度、加速度、转动等）的仪器，
```

File: source/seismology/seismograph.rst [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+几乎与原子间距一样小。大地震（如 2004 年印度尼西亚的苏门答腊地震）产生的地震波可以被全球范围
+的地震仪记录到，振幅高达到几厘米。
+
+数据记录和传输
+--------------
+
+地震学工作者在全球范围布设了许多地震仪，有的布设在地表，有的布设在地下。
+有的地震仪配备了充电设备（如太阳能面板）、网络通讯设备，以便长期稳定地运行，并实时传输数据。
+
+地震发生后，其激发地震波会在地球内部传播。地震波传播至地震仪位置时，地震仪会随即将其记录下来，
```
Comment (seisman) [inline source/seismology/seismograph.rst:35]:
```suggestion
地震发生后，其激发的地震波会在地球内部传播。地震波传播至地震仪位置时，地震仪会随即将其记录下来，
```

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
https://seismo-learn.org/seismology101/seismology/seismograph/

image

Sorry, I didn't view the preview. This image is too low resolution. We should use a high-resolution version.

**Comment by core-man** [comment]
Sorry, I cannot find a high resolution. We may want to plot it ourselves later or find a good alternate.

**Pull request merged**